src/modules/search-time/selector/index.tsx

```diff
@@ -126,10 +126,7 @@ export default function SearchTimeSelector({
 
             <span className={style.option__label}>
               {selectedMode.mode === state && (
-                <motion.span
-                  layoutId="searchTimeSelector"
-                  className={style.option__selected}
-                />
+                <span className={style.option__selected} />
```

Comment:
Don't know if this change is relevant/intended but probably fine! 😄

Reply:
It's intended because the animation got very weird when the global messages were loaded and shown on the page